src/components/Modal/BidVotingModal/BidVotingModal.css

```diff
@@ -0,0 +1,11 @@
+.BidVotingModal .BidVotingModal__Description > p {
```

Comment:
You can use the componentsClassNames prop in the Markdown component to modify all the paragraphs there.